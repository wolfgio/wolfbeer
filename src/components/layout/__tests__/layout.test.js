import React from 'react';
import { create, act } from 'react-test-renderer';

import Layout from '../layout';

describe('<LoadingCard />', () => {
  it('should render without error', () => {
    let root;
    act(() => {
      root = create(<Layout />);
    });

    expect(root.toJSON()).toMatchSnapshot();
  });
});
