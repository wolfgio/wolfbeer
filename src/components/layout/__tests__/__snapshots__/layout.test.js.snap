// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingCard /> should render without error 1`] = `
<div
  className="sc-AykKD ivFVUv"
>
  <header
    className="sc-AykKF kmszHb"
  >
    <div
      className="sc-AykKG gcKfEN"
    >
      <img
        className="sc-AykKH gqLSTR"
        src={
          Object {
            "process": [Function],
          }
        }
      />
      <div
        className="sc-AykKI bMUZia"
      >
        <h1>
          Wolfbeer
        </h1>
        <small>
          Delivery
        </small>
      </div>
    </div>
  </header>
  <div
    className="sc-AykKE cMQUKQ"
  />
  <footer
    className="sc-AykKC bUalAc"
  >
    <p>
      made it by
      <a
        href="https://www.github.com/wolfgio"
        rel="noreferrer noopener"
        target="_blank"
        title="Wolfgio"
      >
         Wolfgio
      </a>
    </p>
    <p
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Icons made by
      <a
        href="https://www.flaticon.com/authors/ddara"
        rel="noreferrer noopener"
        target="_blank"
        title="dDara"
      >
         dDara,
      </a>
      <a
        href="https://www.flaticon.com/authors/srip"
        title="srip"
      >
         srip,
      </a>
      <a
        href="https://www.flaticon.com/authors/freepik"
        title="Freepik"
      >
         Freepik 
      </a>
      and
      <a
        href="https://www.flaticon.com/authors/bqlqn"
        title="bqlqn"
      >
         bqlqn 
      </a>
      from
      <a
        href="https://www.flaticon.com/"
        title="Flaticon"
      >
         www.flaticon.com
      </a>
    </p>
  </footer>
</div>
`;
